packages/order/src/services/__tests__/util/actions/exchanges.ts

```diff
@@ -96,18 +96,20 @@ describe("Order Exchange - Actions", function () {
       actions: actions,
     })
 
-    expect(changes.summary).toEqual({
-      transactionTotal: 0,
+    const sumToJSON = JSON.parse(JSON.stringify(changes.summary))
+    expect(sumToJSON).toEqual({
+      transactionTotal: "0",
       originalOrderTotal: 270,
-      currentOrderTotal: 312.5,
-      temporaryDifference: 62.5,
+      currentOrderTotal: "312.5",
+      temporaryDifference: "62.5",
       futureDifference: 0,
-      futureTemporaryDifference: 0,
-      pendingDifference: 312.5,
-      differenceSum: 42.5,
+      futureTemporaryDifference: "0",
+      pendingDifference: "312.5",
+      differenceSum: "42.5",
     })
```

Comment:
question: This is only for assertion purposes right? Users won't receive these as strings

Reply:
yeah, this is only used internally, so everything that does math operations will handle bignumbers which ends up being "string".
The module responses will always be numeric for these fields.